core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -11887,15 +11919,16 @@ public final String getUserClaimValueWithID(String userID, String claim, String
             }
         }
 
-        // We treat roles claim in special way.
+        // We treat following claims in a special way.
+        List<String> role = null;
```

Comment:
shall we change the name of this variable? it has both role and groups right?

Reply:
roleGroup ?

Reply:
Fixed with d2e0ed4a91a0cb9a37da118d765c8440067ec3be. Renamed to `rolesAndGroups `